integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/LoadTableCommand.scala

```diff
@@ -139,6 +140,15 @@ case class LoadTableCommand(
         carbonLoadModel,
         hadoopConf
       )
+      val operationContext = new OperationContext
+      val loadTablePreExecutionEvent: LoadTablePreExecutionEvent =
+        new LoadTablePreExecutionEvent(sparkSession,
+          null,
+          carbonLoadModel,
+          factPath,
+          dataFrame.isDefined,
+          optionsFinal)
```

Comment:
carbonloadmodel is already getting passed... it there any other property that is getting used from optionsFinal variable?

Reply:
yeah , Iam getting bad records info from optionsFinal for validation purpose in pre event listener.
after validating we are setting to carbonLoadModel, so this is required

Reply:
ok